### PR TITLE
Update Kafka to 2.7.0

### PR DESCRIPTION
**Pull request by gongled**
### What's this PR about

I've updated Kafka to 2.7.0. Nothing special here.

### Known build problems and traps

None.

### TODO's:

- [x] Run [`perfecto`](https://github.com/essentialkaos/perfecto) check on your spec file
- [ ] Write [`bibop`](https://github.com/essentialkaos/bibop) tests
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

**Did you try to build the package with this spec?:** Yes
**Is this ready for review?:** Yes

### CODE REVIEW

File: specs/kafka/SOURCES/checksum.sha512 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ea0522ed4bcc7648c6a22b0fb14d54ef63acf9c0bbcdbb6d8e26904270e4718cba0a7ceabd1b325183157cc978bea9e40d76824ae034ca00b82178ad1cd1b128  2.7.0.tar.gz
```
Comment (andyone) [inline specs/kafka/SOURCES/checksum.sha512]: @gongled `checksum.sha512` is not required for the build process. This file automatically generated by `rpmbuilder` while building packages. Also, this spec doesn't have a macro for checking these checksums.

**Review by andyone — CHANGES_REQUESTED** [review]

**Comment by andyone** [comment]
@gongled Hi! I hope you doing well. Is there any news about this PR?

**Comment by andyone** [comment]
> Files changed: 236

😱

**Comment by gongled** [comment]
@andyone, I removed `checksum.sha512` file. Is there anything else which I can help you with?

**Comment by andyone** [comment]
@gongled I can't merge this PR because it changes a lot of files.

**Comment by gongled** [comment]
> @gongled I can't merge this PR because it changes a lot of files.

I understand. I do not know what happened. I will create yet another pull request properly soon.

**Pull request closed without merging**